src/apiProxy/apiConfigCreators.js

```diff
@@ -30,6 +30,34 @@ export function group(params) {
 }
 
 /**
+ * group communication endpoints
+ *
+ * accepts an `method` property in params
+ */
+export function mugcomm(params) {
+	const path = 'communications';
+	const { urlname, conversationId } = params;
```

Comment:
Yeah that's a valid concern for sure - I think one thing that would help is just renaming `method` to something like `convoInfo` (not a great name, but...). REST principles already give you a lot of the self-documenting advantages:

Without looking at any variable names, it's pretty clear what each endpoint should return

- `/:urlname/conversations` - all conversations in a group
- `/:urlname/conversations/:conversationId` - a single conversation in a group
- `/:urlname/conversations/:conversationId/{followers|suggested_members}` - pretty clear

IMO all that stuff is already self-documented, although I would loooove to be able to use something like typescript/flowjs to declare the supported values for `convoInfo`. In this case, that's the only 'ambiguous' variable, and I think it would be fine to just leave a code comment about what you're expecting, which also gives you a little bit of wiggle room to "automatically" support new types of `convoInfo` endpoints.